bitbucket-slack-server-integration-plugin/src/test/java/com/atlassian/bitbucket/plugins/slack/notification/configuration/DefaultNotificationConfigurationServiceTest.java

```diff
@@ -51,7 +50,7 @@ public void disable_shouldPerformExpectedAction() {
 
         verify(permissionService).validateForRepository(repository, Permission.REPO_ADMIN);
         verify(dao).delete(request);
-        verify(eventPublisher, times(32)).publish(any(RepositoryNotificationDisabledAnalyticEvent.class));
+        verify(eventPublisher, times(33)).publish(any(RepositoryNotificationDisabledAnalyticEvent.class));
```

Comment:
This also isn't an expected consequence of adding PR rescoped event. The plugin shouldn't send analytic event about disabling the notification type, that isn't even allowed to be enabled.
Instead `RESCOPED` value needs to be excluded from `ALL_NOTIFICATION_TYPE_KEYS` set during its initialization via use of [Steam.filter()](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#filter-java.util.function.Predicate-).

Reply:
Can you explain how `PRReScoped` is different from other `PullRequestNotificationTypes`? 
Maybe we should have a separate list of default types that can be enabled?